.github/workflows/manual.yml

```diff
@@ -0,0 +1,36 @@
+on: 
+  workflow_dispatch:
+    inputs:
+      engine_sha:
+        description: 'engine hash'     
+        required: true
+      branch:
+        description: 'branch to be cherry picked onto'     
+        required: false
+        default: 'master'
```

Comment:
In this case, should there be no default?

Reply:
Sure I am good with enforcing a branch parameter